hbase_storage/src/main/java/com/splicemachine/hbase/TransactionsWatcher.java

```diff
@@ -48,14 +48,19 @@
         if (isRestoreMode) return;
 
         long fetchTimestamp = SIDriver.driver().getTimestampSource().currentTimestamp();
-        long oldestActiveTransaction = fetchOldestActiveTransaction();
-        if (oldestActiveTransaction == Long.MAX_VALUE) {
-            lowWatermarkTransaction.set(fetchTimestamp);
-        } else {
-            lowWatermarkTransaction.set(oldestActiveTransaction);
-        }
-        if (LOG.isDebugEnabled()) {
-            LOG.debug(String.format("lowWatermarkTransaction set: %d", lowWatermarkTransaction.get()));
+        try {
+            long oldestActiveTransaction = fetchOldestActiveTransaction();
+            if (oldestActiveTransaction == Long.MAX_VALUE) {
+                lowWatermarkTransaction.set(fetchTimestamp);
```

Comment:
fetchTimestamp can never bee Long.MAX_VALUE? Or do we assume then we can't fix this here anyhow?

Reply:
I don't think that fetchTimestamp can ever reach MAX_VALUE, otherwise, our monotonically increasing timestamp would be broken. @dgomezferro, do you agree?